database/index.js

```diff
@@ -0,0 +1,68 @@
+let mysql = require('mysql');
+var faker = require('faker');
+
+
+let connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: 'password',
+    database: 'slideShowData'
```

Comment:
protect sensitive data like this by gitignoring .env and handling with npm dotenv package

Reply:
password was removed.